Source/WTF/wtf/text/icu/TextBreakIteratorICU.h

```diff
@@ -141,6 +147,54 @@ class TextBreakIteratorICU {
     }
 
 private:
+    // This is temporarily duplicated from LineBreakIteratorPool.
+    // Eventually, LineBreakIteratorPool will be replaced with TextBreakIteratorCache,
+    // and this won't be duplicated any more.
+    static AtomString makeLocaleWithBreakKeyword(const AtomString& locale, LineMode::Behavior behavior)
```

Comment:
Appreciate that it's temporary, but do we need to duplicate? What about defining `LineMode::Behavior` as an alias for `LineBreakIteratorMode`? That should let us call the existing function. Unless there are other behavior changes in this function (aside from the early return optimization) that I'm overlooking.